src/ProcessLauncher.php

```diff
@@ -49,7 +52,7 @@ class ProcessLauncher
     private $runningProcesses = [];
 
     public function __construct(
-        string $command,
+        array $command,
```

Comment:
I would keep `string` here (meaning we still need that `implode()` for the ProcessLauncher) with a TODO to change it to an array when we have the opportunity: otherwise this is a BC break and I'm not keen to tag 2.0 just for this change

Reply:
@theofidry agree, just updated the PR 😊
Thanks again for reviewing!